Gemfile

```diff
@@ -5,10 +5,10 @@ require File.expand_path('../lib/regexp_extensions', FOREMAN_GEMFILE)
 
 source 'https://rubygems.org'
 
-gem 'rails', '3.2.21'
+gem 'rails', '4.0.12'
```

Comment:
rails versions should only be upgraded one minor at a time, we moved one minor from 3.2 to 4.0.